torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5597,6 +5615,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    decorators=(precisionOverride({torch.float16: 1e-2,
                                                   torch.bfloat16: 1e-2}),),
                    safe_casts_outputs=True,
+                   supports_forward_ad=True,
```

Comment:
Are the following three operators tested in this PR?
```
alias
add.Scalar
_sparse_addmm
```
I know the formulas for alias and add.Scalar are really simple, but still would be good to have test coverage

Reply:
I don't see _sparse_addmm added in this PR, it might be the github UI hiding the lines :P
For alias/add.Scalar I agree that it would be good to have those covered, though I'm guessing there aren't any OpInfos yet for that yet?

Reply:
Good point, git blame confused me -- _sparse_addmm is not actually there :)

alias/add.Scalar are only accessible from the C++ API, so they can't really be tested on the Python side...

Reply:
Yes, they cannot be accessed from python :/ 
The two formulas are trivial though so I think its ok to keep them no?

Reply:
I'm fine with merging this without tests for alias and add because those are very simple